installation/security.md

```diff
@@ -1,30 +1,78 @@
 ---
 layout: documentation
-title: Authentication and Encryption through NGINX Reverse Proxy
+title: Securing Communication and Access
 ---
 
 {% include base.html %}
 
-# Running openHAB Behind a Reverse Proxy
+# Securing access to openHAB
+
+openHAB has mainly two ways to be accessed:
+
+1. Through the command line console, which is done through ssh and thus always authenticated and encrypted. You will find all details about this in the [Console documentation](/administration/console).
+2. Through HTTP(S), which we will look at in the following.
+
+{::options toc_levels="2..3"/}
+
+* TOC
+{:toc}
+
+## Encrypted Communication
+
+### Webserver Ports
+
+openHAB has a built-in webserver, which listens on port 8080 for HTTP and 8443 for HTTPS requests. In general, it is advised to use HTTPS communication over HTTP.
+
+The default ports 8080 and 8443 can be changed by setting the environment variables `OPENHAB_HTTP_PORT` resp. `OPENHAB_HTTPS_PORT`. In an apt installation, you would best do this in the file `/etc/defaults/openhab2`.
```

Comment:
No, in a manual setup you have to do it by "setting the environment variables `OPENHAB_HTTP_PORT` resp. `OPENHAB_HTTPS_PORT`" - how you do this, depends on your OS and installation type, so I didn't say anything further.